abcclassroom/config.py

```diff
@@ -25,7 +25,13 @@ def get_github_auth():
     """
     yaml = YAML()
     try:
-        with open(op.expanduser("~/.abc-classroom.tokens.yml")) as f:
+        with open(
+            op.join(op.expanduser("~"), ".abc-classroom.tokens.yml")
```

Comment:
QUESTION: why is this function in config vs github? it doesn't make sense to me here. but i could be convinced otherwise. 

Also a few places conflate expand user with creating a path. this makes testing harder because we want to mock out the specific users home directory. so this needs to be this way to ensure we can easily mock. 

Just make sure i didn't break other things by doing this.

Reply:
@kcranston this is a question about module organization. i'm just curious why some of the github helpers are in config. it's worth just considering organization and i'm very open to whatever we decide i just didn't expect to look for the token helper in config given all other functions are in github. @nkorinek open to your thoughts too!!

Reply:
Only because config.py is where the authorization functions were when I started on the project. I am not opposed to moving all of the git and github stuff into the same place!

Reply:
hi!! ok wonderful. it may make more sense. i started writing tests and realized i had to write tests for config in the github file atleast initially... so perhaps moving it will be good.

Reply:
ok @kcranston you should be able to run the code below. Notice that no matter what i do i can't get to the `FileNotFoundError`. This is because if there's no standard error it fails at the `startswith` check because there is no error message. And otherwise it goes to `RunTimeError` i'm just confused how to hit that last conditional and haven't been able to test on a computer without SSH setup yet. 

```python

import subprocess
import abcclassroom.github as github
import unittest.mock as mock

cmd = ["ssh", "-T", "git@github.com"]
# When i try to mock with a side effect of FileNotFound it doesn't seem
    # to mock correctly. rather it just runs and says no error so the mock
    # isn't working as i think it should
try:
    with mock.patch('subprocess.run') as mock_requests:
        mock_requests.side_effect = FileNotFoundError
        github.check_git_ssh()
        print("no error")
except subprocess.CalledProcessError as e:
    print("called process error")
except FileNotFoundError as e:
    print("File error")

# This raises a runtime as we'd expect the function to do
try:
    with mock.patch('subprocess.run') as mock_requests:
        mock_requests.side_effect = subprocess.CalledProcessError(
            returncode=128, cmd="", stderr="Error here")
        github.check_git_ssh()
        print("no error")
except subprocess.CalledProcessError as e:
    print("called process error")
except FileNotFoundError as e:
    print("File error")
```